fourmolu.yaml

```diff
@@ -0,0 +1,8 @@
+indentation: 2
+comma-style: leading # 'leading' or 'trailing'
+record-brace-space: true # rec {x = 1} vs. rec{x = 1}
+indent-wheres: false
```

Comment:
`indent-wheres: false` didn't do what I expected. Can we change it to `true` before merging this PR?

Reply:
Yes, done!